oauth2_provider/models.py

```diff
@@ -507,7 +507,11 @@ class AbstractIDToken(models.Model):
         related_name="%(app_label)s_%(class)s",
     )
     token = models.TextField(unique=True)
-    jti = models.UUIDField(unique=True, default=uuid.uuid4, editable=False, verbose_name="JWT Token ID")
```

Comment:
@mackosx I think this will work as staging-us won't run 0006 and staging-ca will and hopefully succeed next deploy 🤔.